packages/data/README.md

```diff
@@ -973,6 +973,25 @@ _Returns_
 
 <!-- END TOKEN(Autogenerated API docs) -->
 
+### batch
+
+As a response of `dispatch` calls, WordPress data based applications updates the connected components (Components using `useSelect` or `withSelect`). This update happens in two steps:
```

Comment:
Is there other places where we could use it? Should we make it experimental first?

Reply:
Also, is there any way we could auto-batch? Perhaps we could track all actions in the same call stack and then execute it in the next?

Reply:
Related discussion about "experimental" https://github.com/WordPress/gutenberg/pull/34046#discussion_r689200927

Other places: I don't know maybe at "init" but when I tried it I didn't see any gains, so we'll have to look further. Any ideas?

> Also, is there any way we could auto-batch? Perhaps we could track all actions in the same call stack and then execute it in the next?

The data module works by updating and calling the subscribers synchronously meaning there's no way to auto-batch unless we make the subscription async (setTimeout or something like that), I think it would have a bigger impact and probably some issues here and there in the editor/tests.